tendrl/monitoring_integration/flows/update_dashboard/__init__.py

```diff
@@ -0,0 +1,82 @@
+import json
+import copy
+
+
+from tendrl.monitoring_integration.flows.update_dashboard.alert_utils import get_alert_dashboard
+from tendrl.monitoring_integration.flows.update_dashboard import alert_utils
+from tendrl.monitoring_integration.grafana import dashboard
+from tendrl.commons import flows
+
+
+class UpdateDashboard(flows.BaseFlow):
+
+    def __init__(self):
+        pass
+
+    def run(self):
+        super(UpdateDashboard, self).run()
+        resource_name = str(self.parameters.get("resource_name")).lower()
```

Comment:
Create a new object NS.monitoring.objects.Trigger

attributes of above object

resource_name (type: str)
resource_type (type: str)
action (type: str)
The way it is defined now is invalid, attributes need to have a object associated with them